superset/assets/src/SqlLab/components/ResultSet.jsx

```diff
@@ -207,6 +207,9 @@ export default class ResultSet extends React.PureComponent {
         data = results.data;
       }
       if (data && data.length > 0) {
+        const expandendColumns = results.expanded_columns
```

Comment:
typo: `expandedColumns` instead of `expandendColumns`

Reply:
Thanks, will fix! At least the typos are consistent! :-P